src/converters/RpcHttpConverters.ts

```diff
@@ -25,12 +25,26 @@ export function fromRpcHttp(rpcHttp: rpc.IRpcHttp): RequestProperties {
       query: <Dict<string>>rpcHttp.query,
       params: <Dict<string>>rpcHttp.params,
       body: fromTypedData(<rpc.ITypedData>rpcHttp.body),
-      rawBody: fromTypedData(<rpc.ITypedData>rpcHttp.rawBody, false),
+      rawBody: fromRpcHttpBody(<rpc.ITypedData>rpcHttp.body),
     };
   
     return httpContext;
 }
 
+/**
+ * Converts the provided body from the RPC layer to the appropriate javascript object.
+ * body of type 'byte' is a special case and it's converted to it's utf-8 string representation.
```

Comment:
Can we add a small note about how this is just to avoid breaking changes within major version 2 (v2)?